eav/models.py

```diff
@@ -452,6 +457,18 @@ class Meta:
         verbose_name = _('Attribute')
     )
 
+    @property
+    def value_json(self):
+        if self.value_text:
+            return json.loads(self.value_text)
+        else:
+            return {}
+
+    @value_json.setter
+    def value_json(self, new_value):
+        self.value_text = json.dumps(new_value)
```

Comment:
May be we should check if it is already string then do not convert it?

Reply:
It should be always dict no

Reply:
We can force it to be dict, both are fine with me.